website/content/release-notes/_index.md

```diff
@@ -3,6 +3,58 @@ title: Release Notes
 weight: 8
 ---
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
+  it's now possible to choose to advertise addresses in L2 mode, BGP mode, both or just allocate the IP without advertising it.
```

Comment:
Consider splitting the side effect part into another item, keeping the converter tool in this first item.